src/xcode/ENA/ENA/Source/Scenes/TicketValidation/Cells/TicketValidationNoSupportedCertificateCellModel.swift

```diff
@@ -0,0 +1,31 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import UIKit.UIImage
+
+final class TicketValidationNoSupportedCertificateCellModel {
+
+	// MARK: - Init
+
+	init(
+		serviceProviderRequirementsDescription: String
+	) {
+		self.serviceProviderRequirementsDescription = serviceProviderRequirementsDescription
+	}
+
+	// MARK: - Internal
+
+	var iconImage: UIImage? {
+		return UIImage(imageLiteralResourceName: "Icon_CertificateValidation_Failed")
+	}
+
+	var serviceProviderRequirements: String? {
+		return self.serviceProviderRequirementsDescription
+	}
+
+	// MARK: - Private
+
+	private let serviceProviderRequirementsDescription: String
```

Comment:
```suggestion
 let serviceProviderRequirementsDescription: String
```
use it from internal - the copy of `serviceProviderRequirements` isn't needed